src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -83,6 +85,7 @@ def __init__(
 
         self.display_robot_ids = True
         self.display_speed_lines = True
+        self.display_robot_names = True
```

Comment:
lets make this `False` by default. This layer is usually only helpful when debugging robots in real-life and not that helpful during simulation.